bitex/formatters/bitfinex.py

```diff
@@ -29,32 +29,100 @@ def ticker(self):
 
     def order_book(self):
         """Return namedtuple with given data."""
-        raise NotImplementedError
+        data = self.json()
+        asks = []
+        bids = []
+        if 'bids' in data:  # api version 1
+            for i in data['asks']:
+                asks.append([float(i['price']), float(i['amount'])])
+            for i in data['bids']:
+                bids.append([float(i['price']), float(i['amount'])])
+        else:   # api version 2
+            for i in data:
+                if float(i[2]) > 0:
+                    bids.append([float(i[0]), float(i[2])])
+                else:
+                    asks.append([float(i[0]), -float(i[2])])
+        timestamp = datetime.utcnow()
+        return super(BitfinexFormattedResponse, self).order_book(bids, asks, timestamp)
 
     def trades(self):
         """Return namedtuple with given data."""
-        raise NotImplementedError
+        data = self.json()
+        tradelst = []
+        timestamp = datetime.utcnow()
+        for trade in data:
+            tradelst.append({'id': trade['tid'], 'price': trade['price'],
+                             'qty': trade['amount'], 'time': trade['timestamp']+'000',
+                             'isBuyerMaker': trade['type'] == 'buy', 'isBestMatch': None})
+            # what meaning isBuyerMaker is? if we should remain it in all trades formatter?
+            # raise NotImplementedError
```

Comment:
I believe `isBuyerMaker` states whether or not the buy-side of this trade gets the maker fee or not. I doubt it is available everywhere.

Reply:
I still can not understand what isBuyerMaker mean. Only binance use it. 
Before every trade, of course there's a Maker bid or ask a price, this Maker will get maker fee. And then, the Taker will sell his coin to bid maker, or buy the ask maker's coin, this Taker will get taker fee.
Only binance use 'isBuyerMaker', others use type for sell or buy, these easy to understand, this trade 'buy' the ask price, or 'sell' coin to the bid maker.

okex: {'date': '1523963863', 'date_ms': '1523963863792', 'amount': '0.02204874', 'price': '8113', 'type': 'buy', 'tid': '254541214'}
Bitstamp: {'date': '1523964110', 'tid': '62599526', 'price': '8104.01', 'type': '1', 'amount': '0.28358277'}
Bitfinex: {'timestamp': '1523964119', 'tid': '228108433', 'price': '8098.7', 'amount': '0.1', 'exchange': 'bitfinex', 'type': 'buy'}
Bittrex: {'Id': '48156862', 'TimeStamp': '2018-04-17T11:21:00.653', 'Quantity': '0.02018756', 'Price': '8100.60000000', 'Total': '163.53134853', 'FillType': 'FILL', 'OrderType': 'BUY'}

Reply:
if only `Binance` uses it, this is probably of no relevance in the formatter class. You can safely drop it. Same goes for ``isBestMatch